samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/AutoSubmitFacadeQueries.java

```diff
@@ -72,12 +72,12 @@ else if ( (!acceptLate || retractDate == null) && dueDate != null && currentTime
 				return true;
 			}
 
-			// If it's an "empty" submission don't autosubmit; change status and save (status = 5, hasAutoSubmitRun = true)
+			// If it's an "empty" submission don't autosubmit; change status and save (status = -1, hasAutoSubmitRun = true)
 			// We determine "empty" if it has an attempt date but submitted date is null
 			// Attempt date is populated as soon as student clicks "Begin"; submit date is populated as soon as student makes any progress (next, save, submit)
 			// So if there is an attempt date but no submit date, we can safely assume this is a student who began a quiz and did nothing (either walked away, or logged out immediately)
 			if (adata.getAttemptDate() != null && adata.getSubmittedDate() == null) {
-				adata.setStatus(AssessmentGradingData.NO_SUBMISSION);
+				adata.setStatus(AssessmentGradingData.REMOVED);
```

Comment:
okay @bjones86 much smaller change here. don't send a 0 to the gradebook.

just mark the submission as removed. this allows instructor to manually grade and give the student a score >= 0

Reply:
Nice, I like this much better. Thanks!